src/3rdparty/walkontable/src/overlay/bottomLeftCorner.js

```diff
@@ -24,6 +25,7 @@ export class BottomLeftCornerOverlay extends Overlay {
   constructor(wotInstance) {
     super(wotInstance);
     this.clone = this.makeClone(CLONE_BOTTOM_LEFT_CORNER);
+    this.overlayRoot = null;
```

Comment:
Is really this change needed? I would not change it for consistency with other overlays.